pkg/agent/controller/traceflow/traceflow_controller.go

```diff
@@ -244,17 +261,21 @@ func (c *Controller) syncTraceflow(traceflowName string) error {
 // startTraceflow deploys OVS flow entries for Traceflow and inject packet if current Node
 // is Sender Node.
 func (c *Controller) startTraceflow(tf *opsv1alpha1.Traceflow) error {
-	// Deploy flow entries for traceflow
-	klog.V(2).Infof("Deploy flow entries for Traceflow %s", tf.Name)
-	err := c.ofClient.InstallTraceflowFlows(tf.Status.DataplaneTag)
+	err := c.validateTraceflow(tf)
 	defer func() {
 		if err != nil {
-			c.errorTraceflowCRD(tf, fmt.Sprintf("Node: %s, error: %+v", tf.Name, err))
+			c.errorTraceflowCRD(tf, fmt.Sprintf("Node: %s, error: %+v", c.nodeConfig.Name, err))
 		}
 	}()
 	if err != nil {
 		return err
```

Comment:
can we wrap this error?
```
fmt.Errorf("Error when validating Traceflow request %s: %v", tf.Name, err)
```

Otherwise I feel like the error log message may not have enough context?

Reply:
I think it's enough. The traceflow name will be always attached to error log.
The error log will be:
klog.Errorf("Error syncing Traceflow %s, Aborting. Error: %v", key, err)
err is "service destination requires AntreaProxy feature enabled" here.